{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/radio.jinja

```diff
@@ -0,0 +1,38 @@
+{% from 'forms/attrs.jinja' import form_attrs %}
+
+{% macro radio(name=widget.name, value=widget.value, optgroups=widget.optgroups) %}
+    <div class="radio py-4">
+        <div class="flex gap-4">
```

Comment:
So, by default, horizontal radios, but you can use e.g. `.radio > div { flex-direction: column }` to stack them.

Reply:
Ok, I've added `flex-col` here 👍🏻